redisinsight/ui/src/pages/rdi/pipeline/components/navigation/Navigation.tsx

```diff
@@ -15,7 +17,14 @@ enum RdiPipelineTabs {
   Jobs = PageNames.rdiPipelineJobs,
 }
 
-const defaultNavList = [
+interface INavItem {
+  id: RdiPipelineTabs
+  title: string
+  fileName: string
+  isShowLoader?: boolean
```

Comment:
I was going to comment that this variable name reads strange, but after checking the rest of this codebase it would seem that the `is` prefix is used all over the place to denote a boolean. No action needed. Just wanted to comment on it.